grails-app/migrations/0.9.x/changelog-2024-05-08-1110-rename-picker-to-pickedBy-column.groovy

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog = {
+
+    changeSet(author: "anadolny", id: "080520241110-0", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            and {
+                columnExists(columnName: "picker_id", tableName: "picklist_item")
+                not {
+                    columnExists(columnName: "picked_by_id", tableName: "picklist_item")
+                }
+            }
+        }
+
+        renameColumn(
+                newColumnName: "picked_by_id",
+                oldColumnName: "picker_id",
+                tableName: "picklist_item",
+                columnDataType: "CHAR(38)"
+        )
+    }
+
+    changeSet(author: "anadolny", id: "080520241110-1", objectQuotingStrategy: "LEGACY") {
```

Comment:
I used the `dbm-changelog-to-groovy` command to create a groovy changelog from XML changelog. This is the exact output from this command, so all of the options are probably defaults from XMLs.
objectQuotingStrategy:
![image](https://github.com/openboxes/openboxes/assets/83239466/cac6fbc9-f849-478d-93b1-9fcd97ceea2a)

onSqlOutput:
![image](https://github.com/openboxes/openboxes/assets/83239466/9c7de854-e907-4795-bafc-608e24b34557)
![image](https://github.com/openboxes/openboxes/assets/83239466/be68782f-d671-4b23-8e6a-39469f6aebb8)

Reply:
@jmiranda I was working on a guide for generating changelogs using Liquibase / Database Migration plugin in one of other tickets, or rather I had to try to do it automatically and document the process around it (scope of [OBGM-90](https://pihemr.atlassian.net/browse/OBGM-90) as a part of the [OBPIH-6114](https://pihemr.atlassian.net/browse/OBPIH-6114)). However, I ran into some issues, and it was not working as smooth as expected, and I decided to unblock the OBPIH ticket and I dropped this scope from the feature ticket and move it to the OBGM-90. I think I marked it as in progress, because I wanted to add some more thoughts and comments in there and the ticket itself got lost in the backlog